skimage/metrics/_structural_similarity.py

```diff
@@ -36,6 +37,10 @@ def structural_similarity(
         The side-length of the sliding window used in comparison. Must be an
         odd value. If `gaussian_weights` is True, this is ignored and the
         window size will depend on `sigma`.
+    padding_mode: 'reflect', 'constant', 'nearest', 'mirror', 'wrap' or None, optional
+        The padding mode specifying how the images should be padded. If None, no
+        padding is applied. Padding size is determined by the window size
```

Comment:
```suggestion
        padding is applied. Padding size is determined by the window size, `win_size`,
```
(just to be super clear)